stockflux-components/src/components/ScrollWrapperY/ScrollWrapperY.css

```diff
@@ -0,0 +1,14 @@
+.scroll-wrapper-y {
+  flex: 1 1 auto;
+  position: relative;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  -webkit-transition: 0.6s all;
+  transition: 0.6s all;
+  height: inherit;
```

Comment:
Do we need this inherit style?

Reply:
Yes, the parent sets expected view height, which scrollbar wrapper inherits. This allows us to calculate when the overflow happens, aka when scroll height is higher than client height. If we did not inherit this, then scroll height and client height would be the same.